tests/test_envs.py

```diff
@@ -20,18 +21,6 @@ def matches_list(env_name, patterns):
   return False
 
 
-@pytest.fixture
-def env(env_name):
-  try:
-    env = gym.make(env_name)
-  except gym.error.DependencyNotInstalled as e:  # pragma: no cover
-    if e.args[0].find('mujoco_py') != -1:
-      pytest.skip("Requires `mujoco_py`, which isn't installed.")
-    else:
-      raise
-  return env
-
-
 def rollout(env, actions):
```

Comment:
Looks like this is a duplicate of envs.rollout

Reply:
`assert_equal_rollout` (next function down) doesn't have coverage either

Reply:
thanks, fixed